server/src/main/java/org/opensearch/indices/EhcacheDiskCachingTier.java

```diff
@@ -52,7 +56,7 @@ public class EhcacheDiskCachingTier implements DiskCachingTier<IndicesRequestCac
     private final String diskCacheFP; // the one to use for this node
     private RemovalListener<IndicesRequestCache.Key, BytesReference> removalListener;
     private ExponentiallyWeightedMovingAverage getTimeMillisEWMA;
```

Comment:
Lets also have normal average as well. EWMA might be useful for recent stats and normal will give overall view of get time. As discussed, creating some separate DiskStats might be better. We can move such stats there.